### DISCUSSION TITLE
CHANGELOG causes merge conflicts

### DISCUSSION THREAD

**Discussion by BFH-ktt1** [logisim-evolution/logisim-evolution#1537] [Q&A]
Each time a PR is merged with a change in the CHANGELOG, this causes merge conflicts in the other PR's that also have a change in the CHANGELOG. This is rather annoying, as each time the problem needs to be resolved, and the reviewers need to be again asked to "approve" the PR, as the resolution causes a stale approve.
Would it not be an idea to squash-merge the PR's and than use the PR-titles to automatically produce the changelog when a new release is produced?

**Comment by maehne** [comment]
I am not in favour of automatically generated CHANGELOG entries, as the descriptiveness of PR titles and descriptions varies significantly. Maybe, the CHANGELOG entry should only get discussed as part of the PR and not be part of its patch set. Then, you could add it as a separate commit after merging a PR.
